dlrs/main/classes/OptimizerService.cls

```diff
@@ -147,7 +148,7 @@ global with sharing class OptimizerService {
         notification.Id = NotificationReference.LookupRollupSummaryLogsExist;
         notification.Status = NotificationStatus.Warning;
         notification.Message = 'Check the Lookup Rollup Summary Log tab for errors updating rollups.';
-        notification.MoreInfoURL = 'https://github.com/afawcett/declarative-lookup-rollup-summaries/wiki/Warning:-Lookup-Rollup-Summary-Logs-Exist';
+        notification.MoreInfoURL = 'https://sfdo-community-sprints.github.io/DLRS-Documentation/Issues/#attention-need-to-delete-unused-field-on-dlrs__lookuprollupsummaryscheduleitems__c';
```

Comment:
nice find; did you happen to check the code base for other references like this one?